Resources/doc/reference/architecture.rst

```diff
@@ -429,5 +429,12 @@ of them, you may override the ``configureRoutes`` method::
         }
     }
 
+You can even create any level of nested admin that you want (since 3.x).
```

Comment:
Will a `3.x` reference like this be handled by your release process @Soullivaneuh?

Reply:
It will stay as is

Reply:
How about "You can nest admins as deep as you wish (since 3.x)" instead?

Reply:
Seems good :+1:, but how should I handle the `3.x` reference then?
I had in mind to add something like the **New in version 3.2** text block here: https://symfony.com/doc/current/reference/forms/types/choice.html#choice-loader

Reply:
Just leave it as is, minor upgrades are not a big deal in Sonata, unlike they are in sf, so people don't need that level of precision.

Reply:
Documentation assume you have the last version of the concerned branch: https://sonata-project.org/bundles/admin/3-x/doc/index.html

So no `since 3.x` should be present here.